.github/workflows/icu_ci.yml

```diff
@@ -32,6 +32,27 @@ jobs:
           ant init;
           ant check;
 
+  # Out of source build with gcc 8 and Wextra.
+  gcc-8-Wextra:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: ICU4C with gcc 8 and extra warnings.
+        run: |
```

Comment:
Just curious: What does the pipe symbol do?

Reply:
It starts a multiline string in Yaml.